<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectArduino</class>
 <widget class="QMainWindow" name="ConnectArduino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>136</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>356</width>
    <height>136</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Conexão com o Arduino</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QComboBox" name="comboBox_port"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>    Porta</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_conn">
          <property name="text">
           <string>Conectar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Button_connenct">
          <property name="text">
           <string>Conectar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_baudrate">
          <property name="text">
           <string>baudrate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBox_baud"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_info">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>Não conectado</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
